Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -1385,13 +1426,13 @@ export RGW_POOL_PREFIX=default
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation.
-```console
-kubectl -n rook-ceph-external get sc
-```
-`ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+    ```console
+    kubectl -n rook-ceph-external get sc
+    ```
 
-**NOTE:** For CephFS StorageClass you also need to export `CEPHFS_FS_NAME`, `CEPHFS_POOL_NAME` or you can pass these parameters with a CLI flag (--cephfs-data-pool-name,--cephfs-filesystem-name) while running the python script. For creating ECRBDStorageClass you need to pass --rbd-metadata-ec-pool-name CLI flag or export `RBD_METADATA_EC_POOL_NAME`.
+    !!! note
```

Comment:
You'll be amazed by how much cooler this makes it look when rendered, be sure to check it out when your PR has been merged! :smiley:

Reply:
yup